src/tools/dashboard/src/main.rs

```diff
@@ -4,6 +4,19 @@
 mod dashboard;
```

Comment:
Is this supposed to be here?

Reply:
No. I was just testing the intrinsic functions out and wanted help from the rust analyzer. Removed.